roles/openshift_logging/tasks/oc_apply.yaml

```diff
@@ -18,6 +18,26 @@
   failed_when: "'error' in generation_apply.stderr"
   changed_when: no
```

Comment:
Oh, so this is 'no' here because oc apply will have a rc of 0 even if it didn't actually change anything. So we evaluate the resourceVersion to see if it changed afterwards (lines 2 and then 41)